examples/stable/test/end_to_end/candid_rpc/workspaces/workspaces/ancestor/canisters/old/package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Let's change current and old to something else

Reply:
done